skedge-backend/database/services/database-service.js

```diff
@@ -100,33 +101,62 @@ module.exports = {
     },
 
     insertOneInDatabase: function (objectJSON, collectionName) {
+      return new Promise ((resolve, reject) => {
```

Comment:
If a function returns a promise, it should either resolve or reject that promise. Here, you do neither. Consider resolving and rejecting as you did for createUser a few lines below.